OracleIdentityGovernance/dockerfiles/12.2.1.4.0/idmqs.response

```diff
@@ -0,0 +1,26 @@
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove the copyright and license headers from all `.response` files.

Reply:
done